File: 898-transpose-matrix/transpose-matrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
       vector<vector<int>>ans;
       int n=matrix.size();
       int m=matrix[0].size();
        int i=0;
       for(int col=0;col<m;col++)
       {
           vector<int>temp;
           for(int row=0;row<n;row++)
           {
               temp.push_back(matrix[row][col]);
           }
           ans.push_back(temp);
       } 
       return ans;
    }
};

File: 898-transpose-matrix/transpose-matrix.java
class Solution {
    public int[][] transpose(int[][] matrix) {
        int n=matrix.length;
        int m=matrix[0].length;
        int tps[][]=new int[m][n];
        for(int i=0;i<n;i++){
            for(int j=0;j<m;j++)
            tps[j][i]=matrix[i][j];
        }

        return tps;
    }
}